processes/__init__.py

```diff
@@ -8,6 +8,7 @@
 from .cluster import ClusterProcess
 from .local_development.local_development_setup import LocalDevelopmentSetupProcess
 from .local_development.seed_local_data import SeedLocalDataProcess
+from .local_development.seed_test_data import testSeedLocalDataProcess
```

Comment:
let's change this to SeedTestDataProcess

Reply:
duh